openapi/p4pa-pu-sil.openapi.yaml

```diff
@@ -119,6 +140,48 @@ security:
   - BearerAuth: []
 components:
   schemas:
+    AmountUpdatesLegacyDTO:
```

Comment:
is it possibile to have same model when legacy or not?
who will invoke the API will not be aware of the difference between legacy and not